PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -4,70 +4,79 @@ This document provides information on how to get your repository set up to take
 
 ## Account Setup
 
-First you will need a Router account, and to configure it for SWORDv2 integration.  See your account page in the 
-router for more information.  You need to provide the following:
+Firstly, you will need to contact pubrouter@jisc.ac.uk to create an account. For that, you must provide the matching parameters [template](https://pubrouter.jisc.ac.uk/static/csvtemplate.csv) and the SWORDv2 credentials:
```

Comment:
I suggest replacing 
`and the SWORDv2 credentials:` 
with 
`and the following SWORDv2 credentials:`